component/src/main/java/com/celements/structEditor/classes/SelectTagAutocompleteEditorClass.java

```diff
@@ -0,0 +1,56 @@
+package com.celements.structEditor.classes;
+
+import java.util.List;
+
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+import com.celements.model.classes.AbstractClassDefinition;
+import com.celements.model.classes.fields.BooleanField;
+import com.celements.model.classes.fields.ClassField;
+import com.celements.model.classes.fields.StringField;
+import com.celements.model.classes.fields.list.ComponentListField;
+import com.celements.structEditor.SelectAutocompleteRole;
+
+@Singleton
+@Component(SelectTagAutocompleteEditorClass.CLASS_DEF_HINT)
+public class SelectTagAutocompleteEditorClass extends AbstractClassDefinition implements
+    StructEditorClass {
+
+  public static final String SPACE_NAME = "Celements";
+  public static final String DOC_NAME = "SelectTagAutocompleteEditorClass";
+  public static final String CLASS_DEF_HINT = SPACE_NAME + "." + DOC_NAME;
```

Comment:
add `public static final String CLASS_FN = SPACE_NAME + "." + DOC_NAME;` which has be used by the field builders.
the CLASS_DEF_HINT is the same as our full name.